llvm/tools/sycl-post-link/SpecConstants.cpp

```diff
@@ -735,8 +760,8 @@ PreservedAnalyses SpecConstantsPass::run(Module &M,
   // intrinsic to find its calls and lower them depending on the SetValAtRT
   // setting (see below).
   bool IRModified = false;
-
   LLVMContext &Ctx = M.getContext();
+  std::vector<Function *> SpecConstDeclarations;
```

Comment:
This should most likely be a `SmallVector` instead. See [Programmer's Manual](https://llvm.org/docs/ProgrammersManual.html#sequential-containers-std-vector-std-list-etc):

> Pick the first in this section that will do what you want.